Assets/MXR.SDK/Runtime/Android/MXRAndroidSystem.cs

```diff
@@ -102,6 +107,7 @@ public MXRAndroidSystem() {
             int WIFI_NETWORKS = 1000;
             int WIFI_CONNECTION_STATUS = 3000;
             int RUNTIME_SETTINGS_SUMMARY = 4000;
+            int DEVICE_DATA = 9000;
```

Comment:
This needs to be 19000 - as it stands, initialization will permanently fail because of this.